momaland/learning/discrete/cooperative_momappo.py

```diff
@@ -534,10 +541,15 @@ def save_actor(actor_state):
             save_code=True,
         )
 
-    for i in range(1, 10):  # iterating over the different weights
-        weights = np.array([round(1 - i / 10, 1), round(i / 10, 1)])
-        out.append(train(args, env, weights, rng))
-        print(f"SPS: {(args.total_timesteps * i) / (time.time() - start_time)}")
+    ols = LinearSupport(num_objectives=2, epsilon=0.0001, verbose=True)
+    value = []
+    while not ols.ended():
+        w = ols.next_weight()
+        out = train(args, env, w, rng)
+        actor_state = out["runner_state"][0]
+        _, disc_vec_return = policy_evaluation_mo(actor, actor_state, env=eval_env, num_obj=ols.num_objectives)
+        value.append(disc_vec_return)
+        ols.add_solution(value[-1], w)
```

Comment:
I don't understand the [-1] here?

Reply:
Don't we need to add the last computed evaluation?